src/main/java/org/opentestsystem/ap/imrt/common/model/BaseItem.java

```diff
@@ -93,6 +98,26 @@
 
     private String contentTaskModel = EMPTY;
 
+    private String aslRequired = EMPTY;
+    private String aslProvided = EMPTY;
+    private String brailleRequired = EMPTY;
+    private String brailleProvided = EMPTY;
+    private String ccRequired = EMPTY;
+    private String ccProvided = EMPTY;
+    private String translationRequired = EMPTY;
+    private String translationProvided = EMPTY;
+
+    @OneToMany(mappedBy = "item", fetch = FetchType.EAGER)
+    @Where(clause = "type = 'asl'")
+    private List<Attachment> aslAttachments = new ArrayList<>();
+
+    @OneToMany(mappedBy = "item", fetch = FetchType.EAGER)
```

Comment:
I think all of these should be lazy not eager.  The general search will not need these 90% of the time.

Reply:
Don't we want to always return these in the search results, even though they may not be used by IAT (or whoever the consumer is)?